Server/Quality_of_Life_changer.WebApi/Controllers/CalendarController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Quality_of_Life_changer.Contracts.Interfaces;
+using Quality_of_Life_changer.Contracts.Queries;
+
+namespace Quality_of_Life_changer.WebApi.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class CalendarController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public CalendarController(ICalendarAdapter calendarAdapter, IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    [Route("GetTodayEvents")]
```

Comment:
please follow REST for endpoints naming:
https://restfulapi.net/resource-naming/